shared/src/commonTest/kotlin/me/saket/press/shared/note/FakeNoteRepository.kt

```diff
@@ -56,4 +56,9 @@ class FakeNoteRepository : NoteRepository {
       updateCount++
     }
   }
+
+  override fun markAsArchived(noteUuid: Uuid): Completable {
+   TODO()
+    //We need the instance of a Clock to set the archivedAt property
```

Comment:
This isn't needed. A "fake" repository is used for testing other classes that depend on it, where you don't care what the repository does as long as the class under test is calling the right APIs on the repository. If it were an integration test then you would pass the real repository. 

Does this make sense? I can explain further if you want. You can simply copy the `markAsDeleted()` function here.

Reply:
Thanks for clarifying it makes sense :)